UM-USElab/gradecraft-development
# Removed username column in sample CSV

### Status
**READY**

### Description
* The username column should not be included in the sample students import CSV as it is not used while importing students. Now, the username column has been removed from the sample students import CSV.

One thing to note about this PR is that it removes the username column from csv import, this in turn will remove the ability for umich instructors to enter in only the unique id (username) of a student. They will have to use their full email (including the '@umich.edu')

### Migrations
NO

### Steps to Test or Reproduce
1. Visit the users import page (i.e. /users/import) and download the sample CSV
2. The CSV that is downloaded should not include the username column

### Impacted Areas in Application
* CSV for importing users

======================
Closes #4374
